<launch>
  <node name="slamwareNode"          pkg="slamwareAPP"  type="slamwareNode" output="screen">

  <param name = "ip_addres"                value = "10.16.131.69"/>
  <param name = "angle_compensate"         value = "false"/>
  <param name = "robot_frame"              value = "/base_link"/>
  <param name = "odom_frame"               value = "/odom"/>
  <param name = "laser_frame"              value = "/laser"/>
  <param name = "map_frame"                value = "/map"/>

  <param name = "robot_pose_pub_period"    value = "0.05"/>
  <param name = "scan_pub_period"          value = "0.1"/>
  <param name = "map_pub_period"           value = "0.2"/>
  <param name = "path_pub_period"          value = "0.05"/>

  <param name = "map_size_down_left_x"    value = "-10"/>
  <param name = "map_size_down_left_y"    value = "-10"/>
  <param name = "map_size_width"          value = "25"/>
  <param name = "map_size_height"         value = "25"/>

  <param name = "/cmd_vel"                 value = "/cmd_vel"/>
  <param name = "/move_base_simple/goal"   value = "/move_base_simple/goal"/>
  <param name = "scan"                     value = "scan"/>
  <param name = "odom"                     value = "odom"/>
  <param name = "map"                      value = "map"/>
  <param name = "map_metadata"             value = "map_metadata"/>
  <param name = "global_plan_path"         value = "global_plan_path"/>

  <!-- topic remap /-->
  <remap from = "/cmd_vel"                   to = "/cmd_vel"/>
  <remap from = "/move_base_simple/goal"     to = "/move_base_simple/goal"/>
  <remap from = "scan"                       to = "scan"/>
  <remap from = "odom"                       to = "odom"/>
              =                                 =
  <remap from = "map"                        to = "map"/>
  <remap from = "map_metadata"               to = "map_metadata"/>
  <remap from = "global_plan_path"           to = "global_plan_path"/>

  </node>
</launch>
